bears/yaml/TravisLintBear.py

```diff
@@ -25,6 +28,29 @@ class TravisLintBear:
     CAN_DETECT = {'Formatting', 'Syntax'}
     SEE_MORE = 'https://docs.travis-ci.com/user/travis-lint'
 
+    # IP Address of www.google.com
+    CHECK_CONNECTION_URL = 'http://216.58.218.174'
+    DEFAULT_TIMEOUT = 15
+
+    @classmethod
+    def check_prerequisites(cls):
+        code = cls.get_status_code(
+            cls.CHECK_CONNECTION_URL, cls.DEFAULT_TIMEOUT)
+        if which('travis') is None:  # pragma: no cover
+            return super().check_prerequisites()
```

Comment:
this should be done before every other checks.
-->
```
base_check = super().check_prerequisites():
if base_check is not True:  # <-- you also don't need the `which('travis')` then, because this is done by super().check_prerequisites()
    return base_check

# Now the internet check...
```